src/ImageManipulation.php

```diff
@@ -981,6 +984,8 @@ public function manipulate($variant, $callback)
             $file->setImageBackend($manipulationResult);
         }
 
+        $file->setMutableAttributes($this->attributes);
```

Comment:
This bits just make sure that when we create variant of our file we pass along any attribute attach to the file.